operations/README.md

```diff
@@ -20,6 +20,9 @@ Role Variables
     # Docker version to update to.
     osbs_docker_version: <default not set>
 
+    # Openshift node labels to be set
+    osbs_node_selector_labels: <default not set>
```

Comment:
This should be `osbs_node_labels` thoughout.

Reply:
ok